expenses/models.py

```diff
@@ -322,7 +322,8 @@ def payable():
     @staticmethod
     def accountable(may_account):
         if '*' in may_account:
-            return Expense.objects.exclude(reimbursement=None).filter(verification='').distinct().order_by('expense_date')
+            return Expense.objects.exclude(reimbursement=None).filter(verification='').distinct().order_by(
+                'expense_date')
```

Comment:
Why the linebreak?

Reply:
Auto-formatted that way 😅 The line was probably too long